PhotoViewer/main.py

```diff
@@ -1,34 +1,76 @@
 import tkinter as tk
```

Comment:
Very good, you removed the dummy constants in the code. 
Another thing, the main is getting big and someone reading it cannot understand it (even with the comments)

A good practice is to NOT use comments, and create functions instead like InitDrawingArea(), DrawCircle().... this makes the code clearer and someone can understand your algorithm

Reply:
can this translate to me making functions before the buttons that call the functions in the object or did I not understand this correctly.

Reply:
I think you got the point right. For Example:

lets say I have a function that makes coffee this is more readable:

function MakeCoffee()
{
        BringCup();
        while (waterNotFull):
               AddWater();
        AddCoffee();
 }
THAN:
function MakeCoffee()
{
       # Get Cup
       cup=new Cup()
       a=5 
       cup.get()........
        while (waterNotFull):
               #Add water
               w=new Water()
               a=5 
               w.add()........
       coffee=new Coffee()
       a=5 
       cup.add(coffee)........
 }